src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -174,7 +174,7 @@ object ModelJsonProtocol {
     override def write(input: ESPropertyFields): JsValue = input match {
       case estype: ESType => estype.toJson
       case esaggtype: ESAggregatableType => esaggtype.toJson
-      case essuggtype: ESSuggestField => essuggtype.toJson
+      case esinnerfield: ESInnerField => esinnerfield.toJson
       case _ => throw new SerializationException("unexpected ESProperty type")
     }
 
```

Comment:
change if in read below to be:     
  if (json.asJsObject.fields.contains("fields") && json.asJsObject.fields.get("fields").contains("raw")) {
         ESAggregatableTypeFormat.read(json)